src/components/Accordion/Accordion.test.tsx

```diff
@@ -0,0 +1,28 @@
+import { screen, fireEvent } from '@testing-library/react';
+import { renderWithDeps } from '../../../jest.utils';
+import { Accordion } from './Accordion'; 
+
+describe('<Accordion />', () => {
```

Comment:
Please add an accessibility test here. You can find an example of how to do this in the [updated testing template](https://github.com/Redback-Operations/redback-ui/blob/main/templates/TemplateName.test.tsx).

Reply:
Added the accessibility test in Accordion.test.tsx